source/reference/connection-string.txt

```diff
@@ -202,18 +202,26 @@ DNS Seedlist Connection Format
 .. versionadded:: 3.6
 
 In addition to the standard connection format, MongoDB supports a DNS-constructed
-seedlist. Using DNS to construct the available servers list allows more flexibility
+seed list. Using DNS to construct the available servers list allows more flexibility
 of deployment and the ability to change the servers in rotation without reconfiguring
 clients.
 
-In order to leverage the DNS seedlist, use a connection string prefix of ``mongodb+srv:`` 
-in place of the ``mongodb:`` string above.
+In order to leverage the DNS seed list, use a connection string prefix of
+``mongodb+srv`` rather than the standard ``mongodb``. The ``+srv``
+indicates to the client that the hostname that follows corresponds to a
+DNS SRV record. The driver or :binary:`~bin.mongo` shell will then
+query the DNS for the record to determine which hosts are running the
+:binary:`~bin.mongod` instances.
 
-The ``+srv`` indicates to the mongo client that the hostname that follows corresponds to a 
-DNS SRV record. The client driver will then query the DNS for the record to determine the hosts that
-are running the mongod instances.
+.. note::
+   Use of the +srv connection string modifier automatically sets the
+   tls (or the equivalent ssl) option to true for the connection. You
```

Comment:
Hey Dave, sorry to request edits at this stage, but for tls and ssl, can we please use the `:urioption:` as done in [the master branch version](https://github.com/mongodb/docs/blob/master/source/reference/connection-string.txt#L229) of this file?

I think it's important to link out to those options and they'll get formatted nicely with monospace text.

Reply:
Oh wait, actually `tls` is new in v4.2. I don't think we want this note on v3.6.

Reply:
Ah, I see that the tls option is new in v4.2, so we can't link to it directly from in v3.6. I would suggest either using the double-backtick around `tls` and `ssl` to have them render as monospace, or reword such that we don't mention tls on versions where it's not available.

cc @andf-mongodb